lib/instrumentation/@elastic/elasticsearch.js

```diff
@@ -37,20 +38,29 @@ function queryParser(params) {
   let collection
 
   const suffix = actions[params.method]
-  path.forEach((segment, idx) => {
-    const prev = idx - 1
-    if (segment === '_search') {
-      collection = path?.[prev] || defaultCollection
-      operation = `${segment}`
-    } else if (segment[0] === '_') {
-      collection = path?.[prev] || defaultCollection
-      operation = `${segment}.${suffix}`
+  try {
+    path.forEach((segment, idx) => {
+      const prev = idx - 1
+      let opname
+      if (segment === '_search') {
+        collection = path?.[prev] || defaultCollection
+        operation = `search`
+      } else if (segment[0] === '_') {
+        opname = segment.substring(1)
+        collection = path?.[prev] || defaultCollection
+        operation = `${opname}.${suffix}`
+      }
+    })
+    if (!operation && !collection) {
+      // likely creating an index--no underscore segments
+      collection = path?.[1] || defaultCollection
+      operation = `index.${suffix}`
     }
-  })
-  if (!operation && !collection) {
-    // likely creating an index--no underscore segments
-    collection = path[1]
-    operation = `index.${suffix}`
+  } catch (e) {
```

Comment:
do we want a unit test for this catch behavior? I'm not so sure how to replicate it with a versioned tests so that might be the best